src/frontend/js/components/DashboardCourseList/_styles.scss

```
@@ -0,0 +1,37 @@
+$r-course-glimpse-dashboard-gutter: rem-calc(30px) !default;
+
+.dashboard-course-list {
+  margin-bottom: rem-calc(40px);
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  &__title {
+    font-size: rem-calc(22px);
+    color: r-theme-val(dashboard-course-list, title-color);
+    white-space: nowrap;
+    margin-bottom: rem-calc(10px);
+  }
+
+  //
+  // Course Glimpse in dashboards
+  //
+
+  .dashboard {
+    &__course-glimpse-list {
+      padding-top: 0;
+      margin-top: 0;
+      .course-glimpse-list__content {
+        gap: $r-course-glimpse-dashboard-gutter;
+        .course-glimpse,
+        .course-glimpse__large {
+          @include media-breakpoint-up(lg) {
+            @include sv-flex(1, 0, calc(32% - ($r-course-glimpse-dashboard-gutter / 3)));
```

Not quit sure to understand why you need to divide by 3 ? Why not set `$r-course-glimpse-dashboard-gutter` to `10px`?
```suggestion
            @include sv-flex(1, 0, calc(32% - $r-course-glimpse-dashboard-gutter));
```